docs/src/content/hardhat-network-helpers/docs/reference.md

```diff
@@ -268,3 +268,25 @@ it("test", async function () {
   // use token and exchanges contracts
 })
 ```
+
+## Other helpers
+
+### `dropTransaction(txHash)`
+
+Drops the given transaction hash from the mempool, if it exists.
```

Comment:
```suggestion
Removes the given transaction from the mempool, if it exists.
```

The tx is dropped, not the hash

Reply:
also, `removed` is easier to understand

Reply:
Done in d783f719d113df83fd5b3081d7ac0535ba5e43bf